scripts/src/main/resources/scripts/command/build

```diff
@@ -50,12 +50,12 @@ then
   fi
 else
   echo "Currently not in a buildable project or build system could not be auto-detected"
-  exit -1
+  exit 255
 fi
 
 if [ -n "${1}" ]
 then
-  BUILD_OPTS="${@}"
+  BUILD_OPTS="${*}"
 fi
 echo "${BUILD_CMD} ${BUILD_OPTS}"
-doDevonCommand $BUILD_CMD ${BUILD_OPTS}
+doDevonCommand $BUILD_CMD "${BUILD_OPTS}"
```

Comment:
I think you introduced a bug here. The `BUILD_OPTS` will allow to provide multiple arguments to the build system. So e.g. for maven I could set `BUILD_OPTS=-DskipTests -Pdeploy`. With your change I assume that both different args would be passed as a single parameter and maven will fail. We could test it to get sure...

Reply:
You are completely right. This change is wrong. I'm unsure, why I have did this change, since there is no shellcheck warning for this.